src/commands/argument.js

```diff
@@ -178,7 +184,13 @@ class Argument {
 			}
 
 			// Prompt the user for a new value
-			prompts.push(await msg.reply(stripIndents`
+			this.embed === true ? await msg.replyEmbed(new MessageEmbed().setDescription(stripIndents`
```

Comment:
The description/content should be abstracted into a separate variable - right now it's just being duplicated as the argument to both `reply` and `replyEmbed`

The return value of `await msg.replyEmbed` should also be pushed onto `prompts`

Reply:
I don't know what the default message object is doing in the prompts array or what they're using it for, and if I push the new Message Object obtained from msg#replyEmbed stuff might fuck up since content etc will be reached another way.

Reply:
Possibly, but if you want to make this change I think you'll need to work out why things are pushed into the array.

Not pushing it in could fuck it up too

Reply:
I ended up pushing it